Source/DafnyCore/Reporting.cs

```diff
@@ -43,11 +44,26 @@ public abstract class ErrorReporter {
         IncludeToken includeToken = (IncludeToken)tok;
         Include include = includeToken.Include;
         if (!include.ErrorReported) {
-          Message(source, ErrorLevel.Error, include.tok, "the included file " + tok.Filename + " contains error(s)");
+          Message(source, ErrorLevel.Error, "", include.tok, "the included file " + tok.Filename + " contains error(s)");
           include.ErrorReported = true;
         }
       }
-      Message(source, ErrorLevel.Error, tok, msg);
+      Message(source, ErrorLevel.Error, "", tok, msg);
+    }
+    public void Error(MessageSource source, string errorID, IToken tok, string msg) {
```

Comment:
Can the method on line 38 just call this method with errorID = "" ?

Reply:
Done. In fact the method on line 38 will eventually go away. I just have not cleaned up all the various methods that eventually report a Message until I was sure of the overall structure.